src/terminal/input/terminalInput.cpp

```diff
@@ -674,6 +682,16 @@ bool TerminalInput::HandleKey(const IInputEvent* const pInEvent)
     return false;
 }
 
+bool TerminalInput::HandleFocus(const bool focused)
+{
+    const bool enabled{ GetInputMode(Mode::FocusEvent) };
+    if (enabled)

```

Comment:
Nit: Everywhere else in this class we just test these flags with something like

    if (_inputMode.test(Mode::FocusEvent))